test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -12176,6 +12176,10 @@ def cycles(self, start=None, end=None, firstN=None, lastN=None):
             self.logger.error('error loading ptl.utils.pbs_logutils')
             return None
 
+        if self.id != "default":
+            day = time.strftime("%Y%m%d", time.localtime())
+            self.logfile = os.path.join(self.attributes['sched_log'], day)
+
```

Comment:
This might not be the right place to put this.  the PBSService class has init_logfile_path() which sets self.logfile.  Maybe you should overload it and set self.logfile there.  I mean self.logfile is not correctly set for the multischeds.